scripts/prepare-fvt.sh

```diff
@@ -96,8 +98,17 @@ echo "[${SCRIPT_NAME}] copying plugin to target test folder ..."
 cd "${ROOT_DIR}"
 cp -R .pax/content/. "${FVT_WORKSPACE}/${FVT_PLUGIN_DIR}/"
 cp -R .pax/ascii/. "${FVT_WORKSPACE}/${FVT_PLUGIN_DIR}/"
-cd "${FVT_WORKSPACE}/${FVT_PLUGIN_DIR}/server"
-npm install --production
+cd "${FVT_WORKSPACE}/${FVT_PLUGIN_DIR}"
+echo
+
+################################################################################
+# Explorer UI Server
+echo "[${SCRIPT_NAME}] copying explorer UI server ..."	
+# fetch latest version of explorer ui server
+cd "${FVT_WORKSPACE}/${FVT_UI_SERVER_DIR}"
+npm init -y
+npm install explorer-ui-server --ignore-scripts --registry=https://zowe.jfrog.io/zowe/api/npm/npm-release/
```

Comment:
Please just keep this in mind, we are not using same pax we put into Zowe build for integration test.

Reply:
sure